notification2-examples/notification2-example-client/src/main/java/c8y/example/notification/client/websocket/tootallnate/TooTallNateWebSocketClient.java

```diff
@@ -43,13 +44,23 @@ public void onOpen(ServerHandshake serverHandshake) {
 
         @Override
         public void onMessage(String message) {
+            //Recommended workaround for client issue that can start disconnection if the PONG response are delayed because of busy socket (the RFC 6455 specification does not require the client to send Ping frames if it is constantly receiving messages from the server).
+            ((WebSocketImpl) getConnection()).updateLastPong();
             Notification notification = Notification.parse(message);
```

Comment:
Question: Can these lines outside the `try` also throw?

Reply:
They should not, but just to be on a super safe side I moved everything to the `try` block.